rotary_utils/repair.py

```diff
@@ -34,13 +34,12 @@ def main(args):
     assembly_info_type = 'custom' if args.custom_assembly_info_file is True else 'flye'
     # TODO - improve error handling if a string is provided instead of a real length
     length_thresholds = [int(x) for x in args.length_thresholds.split(',')]
-    cli_tool_settings_dict = {'flye_read_mode': args.flye_read_mode,
-                              'flye_read_error': args.flye_read_error,
-                              'circlator_min_id': args.circlator_min_id,
-                              'circlator_min_length': args.circlator_min_length,
-                              'circlator_ref_end': args.circlator_ref_end,
-                              'circlator_reassemble_end': args.circlator_reassemble_end}
-    threads_mem_mb = int(args.threads_mem * 1024)
+    tool_settings = RepairToolSettings(flye_read_mode=args.flye_read_mode, flye_read_error=args.flye_read_error,
```

Comment:
FYI, I added a new class, `RepairToolSettings`, to replace the `cli_tool_settings_dict` in the old code. The object is easier to document and manage, I think. I also added threads and memory settings into the object.